templates/digital-asset-template/move/sources/launchpad.move

```diff
@@ -213,22 +213,11 @@ module launchpad_addr::launchpad {
                 *option::borrow(&allowlist_end_time),
             );
 
-            let stage_idx = mint_stage::find_mint_stage_index_by_name(collection_obj, stage);
-
-            if (option::is_some(&allowlist_mint_limit_per_addr)) {
-                mint_stage::set_public_stage_max_per_user(
```

Comment:
no need to call `set_public_stage_max_per_user` cuz we set mint limit when calling `add_to_allowlist` below